scene/gui/code_edit.cpp

```diff
@@ -3139,11 +3139,26 @@ void CodeEdit::_update_scroll_selected_line(float p_mouse_y) {
 	code_completion_force_item_center = -1;
 }
 
+// Assumes both the new_options and the code_completion_options are sorted.
+bool CodeEdit::_should_reset_selected_option_for_new_options(const Vector<ScriptLanguage::CodeCompletionOption> &p_new_options) {
```

Comment:
`.cpp` should follow the header ordering so this should be placed after `_filter_code_completion_candidates_impl`